examples/imdb_fasttext.py

```diff
@@ -5,8 +5,9 @@
 Bags of Tricks for Efficient Text Classification
 https://arxiv.org/abs/1607.01759
 
-Can achieve accuracy around 88% after 5 epochs in 70s.
-
+Results on IMDB datasets with uni/bi-gram embeddings:
+    Uni-gram: 0.8813 test accuracy after 5 epochs. 15s/epoch on i7 cpu.
+    Bi-gram : 0.9019 test accuracy after 5 epochs. 5s/epoch on GTX 1080 gpu.
```

Comment:
Yes the batch_size looked small, however increasing the batch_size leads to lower test set accuracy (less exploration I guess).

I let the batch size as it was so we can compare the accuracy when adding n-grams features.